service/dispatcher.js

```diff
@@ -73,11 +73,17 @@ class UserContext extends events.EventEmitter {
 
         console.log(`Context ${this._id}: queued Almond input`, message);
         this._incomingMessageQueue.push({ message, activate });
+        console.log(message);
+        console.log(activate);
     }
 
     close() {
         this._closed = true;
-        this._ws.close();
+        if (this._ws != null) {
```

Comment:
Here and later, use `!==` rather than `!=`

Reply:
Fixed all instances of this, thanks for the advice!